students/drovdahl/codebat/warmup-1/monkey_trouble.py

```diff
@@ -0,0 +1,10 @@
+#! /usr/local/bin/python3
+
+
+def monkey_trouble(a_smile, b_smile):
```

Comment:
Solid work. A more pythonic way to write this would just be:
```
def monkey_trouble(a_smile, b_smile):
    return a_smile == b_smile
```
note that == statements return True/False